src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/LocalAccess.cs

```diff
@@ -424,7 +424,7 @@ public override int Run(InterpretedFrame frame)
 
                 try
                 {
-                    value = Activator.CreateInstance(_type);
+                    value = RuntimeHelpers.GetUninitializedObject(_type);
```

Comment:
@GrabYourPitchforks, does `RuntimeHelpers.GetUnitializedObject(_type)` support `Nullable<T>` or should this method use `Activator.CreateInstance(_type)` or perhaps just `null` if `_type.IsNullableType()`?

Reply:
`GetUninitializedObject` allows `typeof(Nullable<T>)` to be passed in. It extracts the _T_ and returns a boxed `default(T)`.

https://github.com/dotnet/runtime/blob/1011edc50c0ecdaf80bc9b1bc9c802f86583170f/src/coreclr/vm/reflectioninvocation.cpp#L2240-L2242

Reply:
> GetUninitializedObject allows typeof(Nullable<T>) to be passed in. It extracts the T and returns a boxed default(T).

Thanks. For `MutableBox` and `MutableValue`, it seems the value should be `(T?)null` rather than a boxed `(T?)default(T)`.